src/Samples/Toolkit.SampleApp.Maui/Samples/BookmarksViewSample.xaml

```diff
@@ -10,12 +10,10 @@
              x:Class="Toolkit.SampleApp.Maui.Samples.BookmarksViewSample">
     <ContentPage.Resources>
         <DataTemplate x:Key="ItemTemplateOne">
-            <TextCell Text="{Binding Name}" TextColor="Red" />
+            <Label Text="{Binding Name}" TextColor="Red" />
         </DataTemplate>
         <DataTemplate x:Key="ItemTemplateTwo">
-            <ViewCell>
-                <Label BackgroundColor="Red" TextColor="White" Text="{Binding Name}" />
-            </ViewCell>
+            <Label BackgroundColor="Red" TextColor="White" Text="{Binding Name}" />
```

Comment:
Will cells still work or is this a breaking behavior?

Reply:
It think [CollectionView won't support Cell](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/collectionview/populate-data?view=net-maui-8.0#:~:text=a%20DataTemplateSelector.-,Important,an%20exception%20being%20thrown%20because%20CollectionView%20has%20no%20concept%20of%20cells.,-Context%20menus) if we set root of the Template to ViewCell, but we can wrap it inside another element for purpose when we apply template.

Reply:
I tried looking at the options for cells if we can put it in any container. But looks like there is only TableView which takes Cells. But it takes whole lot of space in a row. So it might be breaking change for code using cells.